packages/matplotview.yml

```diff
@@ -0,0 +1,5 @@
+repo: isaacrobinson2000/matplotview
```

Comment:
We should wait to merge for the move to the matplotlib organization.

Reply:
Agreed, I think the project is moved now so I've updated the file for this PR to point to the new location.